docs/content/configuration.md

```diff
@@ -69,6 +69,13 @@ keys:
 caching:
   inter_query_builtin_cache:
     max_size_bytes: 10000000
+
+distributed_tracing:
+  enabled: true
+  address: localhost:4317
+  service_name: opa
+  sample_percentage: 50
+  encryption: off
```

Comment:
🤦 It's not you, it's YAML 1.11 which is used with the config file parser.... anyways, this config will fail to load:

```
error: config error: json: cannot unmarshal bool into Go struct field distributedTracingConfig.encryption of type string 
```
because `off` is interpreted as `false`. To fix the example, we need quotes:
```suggestion
  encryption: "off"
```